inc/customizer/options/layout_single_page.php

```diff
@@ -0,0 +1,492 @@
+<?php
+/**
+ * Single page layout section.
+ *
+ * @package Neve\Customizer\Options
+ */
+
+namespace Neve\Customizer\Options;
+
+use Neve\Customizer\Base_Customizer;
+use Neve\Customizer\Defaults\Single_Post;
+use Neve\Customizer\Types\Control;
+use Neve\Customizer\Types\Section;
+
+/**
+ * Class Layout_Single_Page
+ */
+class Layout_Single_Page extends Base_Customizer {
+	use Single_Post;
+
+	/**
+	 * Customizer section id.
+	 *
+	 * @var string
+	 */
+	private $section = 'neve_single_page_layout';
+
+	/**
+	 * Function that should be extended to add customizer controls.
+	 *
+	 * @return void
+	 */
+	public function add_controls() {
+		$this->section_single_page();
+		$this->add_subsections();
+		$this->header_layout();
+	}
+
+	/**
+	 * Add customize section
+	 */
+	private function section_single_page() {
+		$this->add_section(
+			new Section(
+				$this->section,
+				[
+					'priority' => 45,
+					'title'    => esc_html__( 'Single Page', 'neve' ),
+					'panel'    => 'neve_layout',
+				]
+			)
+		);
+	}
+
+	/**
+	 * Add sections headings.
+	 */
+	private function add_subsections() {
+		$this->add_control(
+			new Control(
+				'neve_page_header_layout_heading',
+				[
+					'sanitize_callback' => 'sanitize_text_field',
+				],
+				[
+					'label'            => esc_html__( 'Header Layout', 'neve' ),
+					'section'          => $this->section,
+					'priority'         => 5,
+					'class'            => 'header_layout-accordion',
+					'expanded'         => true,
+					'accordion'        => true,
+					'controls_to_wrap' => 15,
+				],
+				'Neve\Customizer\Controls\Heading'
+			)
+		);
+	}
+
+	/**
+	 * Add header layout controls.
+	 */
+	private function header_layout() {
```

Comment:
Can we extract most of the common values for the controls between `single_post` and `single_page` and use them from a single source of truth since they are mostly the same only the ID is changed on most of them.